modules/ROOT/partials/module-loading/bundling-vite-es6-npm_config.adoc

```diff
@@ -0,0 +1,27 @@
+. Add the {productname} assets and configuration into the project and provide a target element to initialize the editor. 
++
+Example `+src/index.js+`
++
+[source,js]
+----
+import * as editor from './editor';
+
+const heading = () => {
+  const element = document.createElement('h1');
```

Comment:
Seems like a lot of hula hoops to just get a heading in there we could just add that to the html file instead this isn't really important for how you set this up. We can clean this up later.

Reply:
Agree, ill mark this as todo post 6.8.1 as ill need to re-visit all the other demos which this template originated from.